### ISSUE TITLE
registry issue still not resolved

### ISSUE THREAD

**Issue by pendemdivya** [npm/npm#20863]
I'm opening this issue because:
Previously we have raised an issue https://github.com/npm/npm/issues/20841 but it is still not resolved.
  

What's going wrong?

npm ERR! registry error parsing json
npm http 503 http://registry.npmjs.org/es5-shimify/0.0.1
npm ERR! registry error parsing json
npm http 503 http://registry.npmjs.org/json2ify/0.0.1
npm ERR! registry error parsing json
npm http 503 http://registry.npmjs.org/spine/1.0.6
npm ERR! registry error parsing json

**Comment by legodude17** [comment]
Does this occur if you `npm config set registry https://registry.npmjs.org`?

**Comment by pendemdivya** [comment]
We are just running 
npm install .
npm update

**Comment by legodude17** [comment]
Yes, I know. Just run `npm config set registry https://registry.npmjs.org` and then `npm install`.
